src/main/scala/ai/starlake/schema/handlers/SchemaHandler.scala

```diff
@@ -505,8 +524,9 @@ class SchemaHandler(storage: StorageHandler, cliEnv: Map[String, String] = Map.e
           tranformNode
       val tasksNode = autojobNode.path("tasks").asInstanceOf[ArrayNode]
       for (i <- 0 until tasksNode.size()) {
-        val taskNode = tasksNode.get(i).asInstanceOf[ObjectNode]
+        val taskNode: ObjectNode = tasksNode.get(i).asInstanceOf[ObjectNode]
         YamlSerializer.renameField(taskNode, "dataset", "table")
+        YamlSerializer.renameField(taskNode, "engine", "sqlEngine")
```

Comment:
I would rather rename sqlEngine to engine since the attribute defined if the YAML schema is engine.